package.json

```diff
@@ -75,7 +75,7 @@
     "dev-build": "webpack --progress --mode development",
     "dev-build-test": "npm run dev-build && npm run test",
     "dev-start": "webpack-dev-server --config ./webpack.config.js --mode development",
-    "eslint": "eslint --max-warnings 34 --color -c .eslintrc.js --ext js,jsx ./src",
+    "eslint": "eslint --max-warnings 15 --color -c .eslintrc.js --ext js,jsx ./src",
```

Comment:
Take every opportunity to bump this number downward so it's easier to notice if our linting violations creep upward.